pages/index.vue

```diff
@@ -47,7 +48,7 @@
           </template>
           Read the Blog Post
         </CButton>
-        <CButton green href="https://play.cryptic-game.net" outline text-color="white">
+        <CButton green href="https://play.cryptic-game.net" target="_blank" outline text-color="white">
```

Comment:
add `rel="noopener"` that the page from which you are redirected can't track the website from which you are